### ISSUE TITLE
MemoryView can't find debug-tracker-vscode when used on a remote

### ISSUE THREAD

**Issue by rubenstar** [mcu-debug/debug-tracker-vscode#1]
I'm trying to use Cortex-Debug, MemoryView and debug-tracker-vscode on a remote server I connect to through the "Remote - SSH" addon. Even though everything is installed on the remote side, a memory view command fails with the following message:

Cannot activate the 'MemoryView' extension because it depends on the 'debug-tracker-vscode' extension, which is not loaded. Would you like to reload the window to load the extension?

Reloading window, or restarting VSCode won't change the issue.

**Comment by haneefdm** [comment]
Thank you for reporting this issue.

Could you tell me which version of MemoryView is installed?

Did you request memory from Cortex-Debug or from MemoryView. You can do either.

Is there an extension called `debug-tracker-vscode` installed? It should automatically install when you install MemoryView

I am thinking that memoryview and debug-tracker-vscod should actually be on the UI side and not on the remote side. Bit confusing for me because VSCode puts Cortex-Debug on the Remote side as reported by others. Is there a way to tell what is running on the Remote side and what is running on the UI side.

**Comment by haneefdm** [comment]
I have a feeling, we are being classified as a 'UI' extension and not a 'Workspace' extension.

https://code.visualstudio.com/api/advanced-topics/remote-extensions#architecture-and-extension-kinds

I have zero experience with 'Remote - SSH' or anything Remote really, but we should make it work.

**Comment by rubenstar** [comment]
> Could you tell me which version of MemoryView is installed?

v0.0.12

> Did you request memory from Cortex-Debug or from MemoryView. You can do either.

Both, same result

> Is there an extension called debug-tracker-vscode installed? 

Yes. 

> I am thinking that memoryview and debug-tracker-vscod should actually be on the UI side and not on the remote side. 
> I have a feeling, we are being classified as a 'UI' extension and not a 'Workspace' extension.

That sounds like it could be the issue. Both Cortex-Debug and MemoryView need to be installed specifically on the remote side to work.
![image](https://user-images.githubusercontent.com/25745502/195083928-97aef534-4bcd-49e7-9344-2d769e082a54.png)
![image](https://user-images.githubusercontent.com/25745502/195083797-f446f452-ab65-4a22-b7cf-a118e46f3264.png)

For debug-tracker-vscode I do not get such a prompt. Installing it on the local machine (automatically with ViewMemory) will list it for both local machine and host as "enabled globally". 
![image](https://user-images.githubusercontent.com/25745502/195082324-934b7849-947f-4f81-aa97-9c339346a2b6.png)

Using MemoryView locally works, but on the remote workspace I'll get the original error. It seems like "enabled globally" is a little misleading.

Thanks a lot for your continuous effort! Its a great tool!

Edit: Maybe worth mentioning that having Cortex-Debug on the remote and ViewMemory/debug-tracker-vscode on the host won't work. Cortex-Debug demands I install ViewMemory on the remote too.

**Comment by haneefdm** [comment]
Technically, the debug-tracker-vscode can exist anywhere, but the way VSCode works, it is a chain reaction. If cortex-debug is running remotely (and it has to) so do the other two MemoryView does need to run where your workspace lives as it saves view information.

One problem I found was that the tracker was erroneously marked as 'ui' only. It should have been marked with both 'workspace' and 'ui' with the former being the preference. If this does not work, I will have to mark it as 'workspace' only. We will see.

I published updated versions of both the MemoryView and the Tracker but it may take a few minutes. The servers must be busy because it is taking forever. I will post again when I get a notification,. It never took this long.
